example/providers.tf

```diff
@@ -9,3 +9,19 @@ provider "nomad" {
   # Add a secret_id if ACLs are enabled in nomad
   secret_id = var.nomad_acl ? data.vault_generic_secret.nomad_secret_id[0].data.secret_id : null
 }
+
+provider "vault" {
+  address = "http://127.0.0.1:8200"
+  token = "master"
```

Comment:
Could these be env variables?

Reply:
In theory - yes. But it is an example, so I would prefer to have it explicit

Reply:
I agree. Good point!